app/api/v2/utils/database.py

```diff
@@ -0,0 +1,56 @@
+import psycopg2,os
+from Instance.config import app_config
+
+
+config = os.getenv("FLASK_ENV")
+
+def init_db():
+    try:
+        if config == "development":
+            db_url = os.getenv("DATABASE_URL")
+        else:
+            db_url="dbname='testingdb' host='127.0.0.1' port='5433' user='postgres' password='12345'"
```

Comment:
I think you can also create a `TEST_DATABASE_URL` just like how you have created the development `DATABASE_URL` and get it from the env.